backend-tests/README.md

```diff
@@ -27,7 +28,8 @@ By default, this tests the wasm file in
 
     ../target/wasm32-unknown-unknown/release/internet_identity.wasm
 
-to use a different one, pass the `--wasm` flag to `backend-tests`
+to use a different one, pass the `--wasm` flag to `backend-tests`. You will
+need to build with `USE_DUMMY_CAPTCHA` as the tests use a preset CAPTCHA value.
```

Comment:
Perhaps clarify that this is an environment variable?

Reply:
Not  important though.  I think it is reasonably guessable.  Feel free to deploy without this change.

Reply:
mh, at least I should mention what value to give `USE_DUMMY_CAPTCHA`! will update.